.github/workflows/ci.yml

```diff
@@ -62,8 +62,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-22.04]
-        cabal: ["3.10.1.0"]
-        ghc: ["9.2.8", "9.4.8", "9.6.3"]
+        cabal: ["3.12.1.0"]
+        ghc: ["9.4.8", "9.6.3", "9.8.4"]
```

Comment:
It might be worth upgrading to the latest minor versions of each of these GHC releases (in particular, 9.4.8, 9.6.6, and 9.8.4) while we're at it.